test_files/declare/declare.d.ts

```diff
@@ -51,6 +51,11 @@ declare module "DeclareTest-QuotedModule" {
 declare var declareGlobalVar: number;
 declare function declareGlobalFunction(x: string): number;
 
+/**
+ * @suppress {duplicate}
+ */
+declare var declareGlobalVarWithSuppress: number;
```

Comment:
My recollection is that even with suppress duplicate, if the types don't match it's an unsuppressable compiler error.  So usually using suppress won't save you.

Also, I experimented on go/jscomp-full and it appears a file-level suppress does work if the first block is tagged @externs.  Maybe we should just always emit that?